package.json

```diff
@@ -77,7 +77,7 @@
 		"cross-env": "^7.0.3",
 		"css-loader": "^6.0.0",
 		"ejs": "^3.1.10",
-		"electron": "35.0.0",
+		"electron": "^35.0.0",
```

Comment:
Is this change intentional?
I think it makes sense to update Electron patch versions, but it would be better to do it in a beta release first.